src/services/api/client.js

```diff
@@ -14,19 +14,26 @@ const apiUrl = (requestPath) => {
 
 export default {
   request: (requestPath = '', data = {}, method = 'GET') => {
-    if (method === 'GET') {
-      const requestParams = queryParams.encode(data);
+    let payload = {};
+    let requestParams = queryParams.encode(data);
```

Comment:
Maybe use `const` if the value does not change. But I'm not too strict on this 😏